quesma/telemetry/phone_home.go

```diff
@@ -296,6 +296,63 @@ where active
 	return nil
 }
 
+func (a *agent) collectClickHouseTableSizes(ctx context.Context) (map[string]int64, error) {
+	tablesWithSizes, err := a.getTableSizes(a.ctx)
+	if err != nil {
+		logger.WarnWithCtx(ctx).Msgf("Error getting table sizes from clickhouse: %v", err)
+		return nil, err
+	}
+	return tablesWithSizes, nil
+}
+
+func (a *agent) getDbInfo() string {
```

Comment:
I'm debating whether we want to collect that. Maybe we should hash some of that, in particular URL may be sensitive in some env.